README.md

```diff
@@ -283,10 +283,11 @@ And finally, you are ready to go! Or at least you should be...
 
 ### Junior Contributors
 
-
+**Bruno Alvarez** | [contact me][brunoalvarez5] | La vida es una Tombola
```

Comment:
It's almost there, just a little change and it will be perfect.

Instead of  `contact me `  change by your  `nickname`  of GitHub and would be it!